backend/test/integrated/issue.js

```diff
@@ -92,91 +96,78 @@ describe("Issues", function () {
+			const issue = {"name":"Issue test", ...cloneDeep(baseIssue)};
+			issue.viewpoint.screenshot = pngBase64;
 
-							return done(err);
-						});
-				},
+			let res = await agent.post(`/${username}/${model}/issues`)
+						.send(issue)
+						.expect(200)
 
-				function(done) {
-					agent.get(`/${username}/${model}/issues/${issueId}`).expect(200, function(err , res) {
+			let issueId = res.body._id;
 
-						expect(res.body.name).to.equal(issue.name);
-						expect(res.body.scale).to.equal(issue.scale);
-						expect(res.body.status).to.equal(issue.status);
-						expect(res.body.topic_type).to.equal(issue.topic_type);
-						expect(res.body.priority).to.equal(issue.priority);
-						expect(res.body.creator_role).to.equal(issue.creator_role);
-						expect(res.body.assigned_roles).to.deep.equal(issue.assigned_roles);
-						expect(res.body.viewpoint.up).to.deep.equal(issue.viewpoint.up);
-						expect(res.body.viewpoint.position).to.deep.equal(issue.viewpoint.position);
-						expect(res.body.viewpoint.look_at).to.deep.equal(issue.viewpoint.look_at);
-						expect(res.body.viewpoint.view_dir).to.deep.equal(issue.viewpoint.view_dir);
-						expect(res.body.viewpoint.right).to.deep.equal(issue.viewpoint.right);
-						expect(res.body.viewpoint.unityHeight).to.equal(issue.viewpoint.unityHeight);
-						expect(res.body.viewpoint.fov).to.equal(issue.viewpoint.fov);
-						expect(res.body.viewpoint.aspect_ratio).to.equal(issue.viewpoint.aspect_ratio);
-						expect(res.body.viewpoint.far).to.equal(issue.viewpoint.far);
-						expect(res.body.viewpoint.near).to.equal(issue.viewpoint.near);
+			res = await agent.get(`/${username}/${model}/issues/${issueId}`).expect(200);
 
-						return done(err);
-					});
-				}
-			], done);
+			expect(res.body.viewpoint.screenshot).to.equal(`${username}/${model}/issues/${issueId}/viewpoints/${res.body.viewpoint.guid}/screenshot.png`);
 		});
 
-		it("with screenshot should succeed", function(done) {
-			const issue = Object.assign({"name":"Issue test"}, baseIssue);
-			issue.viewpoint.screenshot = pngBase64;
-
-			let issueId;
+		it("with screenshot using the wrong file format should fail", async function() {
+			const issue ={"name":"Wrong file issue", ...cloneDeep(baseIssue)};
+			issue.viewpoint.screenshot = pdfBase64;
 
-			async.series([
-				function(done) {
-					agent.post(`/${username}/${model}/issues`)
-						.send(issue)
-						.expect(200 , function(err, res) {
-
-							issueId = res.body._id;
-							return done(err);
-						});
-				},
-				function(done) {
-					agent.get(`/${username}/${model}/issues/${issueId}`).expect(200, function(err , res) {
-						expect(res.body.viewpoint.screenshot).to.equal(`${username}/${model}/issues/${issueId}/viewpoints/${res.body.viewpoint.guid}/screenshot.png`);
-						return done(err);
-					});
-				}
-			], done);
+			await agent.post(`/${username}/${model}/issues`)
+					.send(issue)
+					.expect(responseCodes.FILE_FORMAT_NOT_SUPPORTED.status);
```

Comment:
we should check the body of the response to make sure the message is right as well.